src/Bonsai.ML.Torch/ObserveWithGradientTracking.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Reactive;
+using System.Reactive.Linq;
+using TorchSharp;
+using Bonsai;
+using System.ComponentModel;
+
+/// <summary>
+/// This operator ensures that all tensor operations within the observable sequence are executed with gradient tracking enabled.
+/// </summary>
+[Combinator]
+[Description("Ensures that all tensor operations within the observable sequence are executed with gradient tracking enabled.")]
```

Comment:
```suggestion
/// Represents an operator that ensures all tensor operations within the observable sequence are executed with gradient tracking enabled.
/// </summary>
[Combinator]
[Description("Ensures all tensor operations within the observable sequence are executed with gradient tracking enabled.")]
```

Reviewing wording for consistency with above comments.